Exercise_9/src/MyFramework/services/historyService.js

```diff
@@ -3,6 +3,14 @@ const historyService = {
   addToHistory: ((url) => {
     window.history.pushState({}, 'Route', url);
   }),
+  createHistoryNavigation: (appInstance) => {
+    window.onpopstate = () => {
+      const currentUrl = window.location.pathname;
+      if (currentUrl === '/') {
```

Comment:
I still don't understand why do you need this condition. For what? What if it is not "/"?

Reply:
try to remove this condition and then open file, click few times on sign-in button, and then try to click back button on the browser, it's not working as should